functions/zoom-downloader.py

```diff
@@ -152,25 +190,32 @@ def series_id_from_schedule(self):
             logger.info(schedule)
 
         zoom_time = self.created.astimezone(timezone(LOCAL_TIME_ZONE))
-        weekdays = ['M', 'T', 'W', 'R', 'F']
+        weekdays = {'M': 'Mondays',
+                    'T': 'Tuesdays',
+                    'W': 'Wednesdays',
+                    'R': 'Thursdays',
+                    'F': 'Fridays'}
         if zoom_time.weekday() > 4:
             logger.debug("Meeting occurred on a weekend.")
             return None
-        elif weekdays[zoom_time.weekday()] not in schedule['Days']:
-            logger.debug("No opencast recording scheduled for this day of the week.")
+        letter = list(weekdays.keys())[zoom_time.weekday()]
+        if letter not in schedule['Days']:
+            logger.debug("No opencast recording scheduled on {}."
+                         .format(weekdays[letter]))
```

Comment:
This doesn't  matter, I just wanted to print out the day of the week so the log reader doesn't have to go looking for it.